spec/defines/resource_server_spec.rb

```diff
@@ -904,18 +904,6 @@
         it { is_expected.to contain_nginx__resource__location("#{title}-default").with_ssl_only(true) }
       end
 
-      context 'SSL cert missing' do
```

Comment:
yes. In general we need more tests. but this is more of a global issue on all of our modules.